### ISSUE TITLE
iOS: Unable to write unicode characters in accessibility id

### ISSUE THREAD

**Issue by chanugithub** [appium/appium#9989]
* Appium version (or git revision) : 1.7.2 beta
* Desktop OS/version used to run Appium: 10.12.6
* Mobile platform/version under test: iPhone X (11.2.1)

## The problem

I am unable to write unicode characters in accessibility id. In native application, i am clicking on a link which open web page and need to hit the back key which goes back to native app. Now the iPhone X (v11.2) is having unicode character which is not allowing me to write via accessibility id. For example: Now the back key resembles like  '◀︎ Google'.  so ◀︎ symbol unicode value is U+25C0.
I am getting error when i try to write accessibility id like -- iDriver.findElementByAccessibilityId("(U+25C0 Google");

what is the correct way to write unicode characters or does appium actually support unicode characters?

**Comment by mykola-mokhnach** [comment]
have you tried to simply copy the element name from page source output? Also, it's possible to use such helper functions, like CONTAINS, BEGINSWITH and ENDSWITH in predicate locators.

It would be also useful to have the full server log.

**Comment by chanugithub** [comment]
@mykola-mokhnach Thanks for reply, yes i am able to execute with helper function CONTAINS i.e iDriver.findElementByIosNsPredicate(("label CONTAINS 'Google'"));

I observe for android there is support for some unicode keyboard keys in uiautomator2 https://github.com/appium/appium-uiautomator2-driver/blob/master/test/functional/commands/keyboard/keyboard-e2e-specs.js#L118

It would be nice if there is something for iOS in future, anyways i would close this issue
